Session.vim

```diff
@@ -0,0 +1,87 @@
+let SessionLoad = 1
```

Comment:
git ignore this file :)

Reply:
yeah sure. Just a work-in-progress thing when I move between my machines :)